logger/log_writter.go

```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+type LogWritter struct {
```

Comment:
Can we rename this to `LogWriter`? I think thats what you were going for unless I'm missing something (filename too).

Reply:
sure, I fixed it 👍